src/serializableStateInvariantMiddleware.ts

```diff
@@ -25,10 +33,10 @@ const NON_SERIALIZABLE_ACTION_MESSAGE = [
 ].join('\n')
 
 export function findNonSerializableValue(
-  obj,
-  path = [],
-  isSerializable = isPlain
-) {
+  obj: any,
+  path: string[] = [],
+  isSerializable: (value: any) => boolean = isPlain
+): any {
```

Comment:
`obj: unknown`, `path: ReadonlyArray<string> = []`, and you don't need an annotation for `isSerializable` (the correct annotation would be `: typeof isPlain` anyway).

(implementation comment: replace `for..in + hasOwnProperty` with `Object.keys`)